week-3/refactor/format-as-12-hours.js

```diff
@@ -4,3 +4,29 @@
 // Store this expression in a variable and reference it twice in the function in the correct place
 
 // Explain why it makes more sense to store this expression in a variable
+//after storing the expression in variable, there is no need to rewrite the expression
+function formatAs12HourClock(time) {
+  const clockHourIn12 = Number(time.slice(0, 2));
+  if (clockHourIn12 > 12) {
+    return `${clockHourIn12 - 12}:00 pm`;
```

Comment:
if a value like `18:46` is send in here, the result according to your code will be `6:00pm` which is wrong. Also is the time send is `09:00` the result will be `09:00` without the `am` appended. You need to take care of these cases